packages/colibri/src/config/config_web.ts

```diff
@@ -616,7 +616,17 @@ export const WEB_CONFIG = `
               </select>
             </div>
           
-          
+            <div class="mb-3">
```

Comment:
Have you used the Python script or manually?

Reply:
This part was done by hand. was there a python script that can be used for these updates?

Reply:
updated packages/colibri/src/config/helpers/configs/documentation.yml to get it to update correctly with running the python script.